### DISCUSSION TITLE
Internal error in login process?

### DISCUSSION THREAD

**Discussion by RobotsAreCrazy** [cockpit-project/cockpit#17056] [Q&A] [answered]
### Explain what happens

1.go to gui url 
2. get the error Internal error in login process
3. 

Any idea what might cause it and a possible fix?

Thankyou

### Version of Cockpit

Can't get the version info just that error

**Comment by martinpitt** [comment, marked as answer]
I suppose this applies to your followup in https://github.com/cockpit-project/cockpit/issues/9527#issuecomment-1052019944 .

Please install a correct PAM file for your system, supposedly Arch? https://github.com/cockpit-project/cockpit/blob/main/tools/arch/cockpit.pam

Otherwise, please tell us your OS, cockpit version, how you installed, etc (that form is there for a reason :wink: )

**Reply by RobotsAreCrazy** [reply]
Hi Martin, built from source, latest version from 3 days ago used configure /etc p.s. not exact cmd i used disabled pcp etc it built fine, installed it from /tools/arch/ pacman -u  cockpit-264-1-aarch64.tar.zst  

i'm pretty sure its to do with that cockpit.pam config  contents

**Reply by RobotsAreCrazy** [reply]
I did do some messing on with what was in it....now on trying to login on root or my user account i get message 
Authentication failed: internal-error: Module is unknown
on manjaro-arm on rpi4 uptodate etc

**Reply by RobotsAreCrazy** [reply]
and before you say why didn't i just use default aur package, it is older and i liked the challenge to build and get it running and lastly it's helping me learn arch system

**Reply by martinpitt** [reply]
That is a perfectly good reason :wink: I just wanted to confirm it's the same scenario as in #9527.

"module is unknown" sounds like /etc/pam.d/cockpit is trying to invoke a nonexisting PAM module. `system-remote-login` really should exist on Arch, [cockpit.pam](https://github.com/cockpit-project/cockpit/blob/main/tools/arch/cockpit.pam) isn't doing anything weird. Can you please start `sudo journalctl -f`, try to log into cockpit, and then copy&paste the journal output here?

**Reply by RobotsAreCrazy** [reply]
bit of another weird one, cockpit 264-1 cockpit-podman 42-1 cockpit-test 264-1  podman is not showing up even though service is on, but then turns itself off etc some strange reason it is not working with podman...

**Reply by martinpitt** [reply]
That description is a bit vague, but it sounds like `podman.socket` and/or `podman.service` is crashing? (Either for the system or the user)

**Reply by RobotsAreCrazy** [reply]
Hi Martin, what are my options to try and fix it?

i did install dev 4.00 podman from aur, but still the same...

**Reply by martinpitt** [reply]
Sorry, with that little information I don't even know what's broken. Please check the journal for podman error messages, and perhaps send a screenshot what's wrong.

**Reply by RobotsAreCrazy** [reply]
Hi, how do i check journal? at this point i feel like uninstalling podman, cockpit-podman and cockpit 2.64-1

**Reply by martinpitt** [reply]
As I mentioned above: `sudo journalctl -f` for following current logs; you can also do e. g. `sudo journalctl -b` to get the whole logs for the current boot, etc. See manpage.

**Comment by RobotsAreCrazy** [comment]
Also going in to app and to podman shows not found...could there be a incompatibility between old podman version 3.4.4-1 on aur and your newer build of 2.64.1? 

Only way to maybe know is if i downgraded to 2.63 currently on aur and if it then works to show and have podman usable then we know?

or maybe i just need to build podman to newer version from source....

**Reply by jelly** [reply]
Please provide a screenshot where it shows "podman not found".

**Reply by RobotsAreCrazy** [reply]
Applications --> cockpit-podman then shows not found

**Comment by Microsvuln** [comment]
I can truly confirm that one of the situations that cockpit users see **Internal error in login process** error message is when the server disk space is full (out of space) . feel free to test it. 

Thanks.

**Reply by Watever44** [reply]
How should I resolve a full disk ? if I can't log to clean it ?

**Reply by Microsvuln** [reply]
use SSH to login into the server and clean the /tmp files related to cockpit and then you will be able to login via cockpit interface.

**Reply by zouchanglin** [reply]
Thank you very much. It is because the disk is full that I cannot log in！

**Reply by zouchanglin** [reply]
install debian in my main disk, but it only own 32G, docker images used 6G……

**Comment by kivison** [comment]
Just a curveball but Ive just had this issue on a server I don't go on a lot and the disk was full, nothing to do with anything else..
